File: Front/src/app/models/activiteLite.ts
import { AdherentLite } from "./adherentLite";

export class ActiviteLite {

  id: number;
  nom: string;

  horaire: string;
  lien: string;
  salle: string;
  adherents: AdherentLite[];

  constructor(){
    this.id = 0;
    this.nom = "";
       this.horaire = "";
    this.lien = "";
    this.salle = "";
    this.adherents = [];
   
  }
}

File: app/src/main/java/com/wild/corp/adhesion/models/ActiviteLite.java
package com.wild.corp.adhesion.models;

import lombok.*;

import java.util.HashSet;
import java.util.Set;

@Getter
@Setter
@ToString
@Builder
@AllArgsConstructor
@RequiredArgsConstructor
@EqualsAndHashCode(of = {"id"})
public class ActiviteLite {

    private Long id;
    private String nom;
    private String lien;
    private String horaire;
    private String salle;
    private String groupe;
    private Set<AdherentLite> adherents = new HashSet<>();

}
